dso-l2/src/main/java/com/tc/l2/state/SafeModeConsistencyManagerImpl.java

```diff
@@ -0,0 +1,114 @@
+package com.tc.l2.state;
```

Comment:
Rather than having two separate implementations, can a delegate model be used where common behavior can be pulled into the SafeModeConsistencyManagerImpl (basically the case where a server is transitioning from START->ACTIVE) then delegated to ConsistencyManagerImpl if consistency mode is selected or nothing if availability mode is selected.

Reply:
Would be nice too if the server node tracking (GroupListener) could be factored out and shared as well but that can be future cleanup.